src/create-model.jl

```diff
@@ -205,7 +205,10 @@ function create_model(graph, representative_periods, dataframes; write_lp_file =
     ## Sets unpacking
     A = labels(graph) |> collect
     F = edge_labels(graph) |> collect
-    filter_assets(key, value) = filter(a -> getfield(graph[a], key) == value, A)
+    filter_assets(key, value) = filter(
+        a -> getfield(graph[a], key) === missing ? false : getfield(graph[a], key) == value,
+        A,
+    )
```

Comment:
There are two slightly cleaner ways to do this:

```julia
a -> !ismissing(getfield(graph[a], key)) && getfield(graph[a], key) == value
```
So you check using `ismissing`, which simplifies `=== missing`, and then use `&&` which [short-circuits](https://en.wikipedia.org/wiki/Short-circuit_evaluation) to avoid the ternary operator.

Or
```julia
a -> coalesce(getfield(graph[a], key) == value, false)
```
Which will compute `getfield(graph[a], key) == value` and then `coalesce` will find the first non-missing argument. It is either `true` or `false` from the first argument, or `false` from the second.
Given the size of `A` and the number of times that we will be calling these operations, the cost is negligible, so I think it boils down to what looks better. 

PS. Code not tested

Reply:
Great! I have implemented the first option 👍